contracts/examples/AyiiProduct.sol

```diff
@@ -42,7 +42,8 @@ contract AyiiProduct is
     uint256 private _oracleId;
 
     mapping(bytes32 /* riskId */ => Risk) private _risks;
-    mapping(bytes32 /* riskId */ => bytes32 [] /*policyIds*/) private _policies;
+    mapping(bytes32 /* riskId */ => bytes32 [] /* policyIds */) private _policies;
+    mapping(bytes32 /* policyId */ => bytes32 /* riskId */) private _riskIdByPolicy;
```

Comment:
@matthiaszimmermann I think `_policies` and `_riskIdByPolicy` can get quite large with time. We should add a mechanism to clean those. Any idea how this could be best achieved?

Reply:
missed the .pop() to clean out the list again.